docs/querying/api.md

```diff
@@ -892,6 +892,10 @@ The following endpoint returns various cardinality statistics about the Promethe
 ```
 GET /api/v1/status/tsdb
 ```
+- **numSeries**: This provides the number of series in the tsdb.
+- **chunkCount**: This provides the number of chunks in the tsdb.
+- **minTime**: This provides the current minimum time in the tsdb.
```

Comment:
How about mentioning the unit (milliseconds)? So "time" -> "timestamp in milliseconds".